### DISCUSSION TITLE
Version 0.7.2

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#679] [Announcements]
Good day! Machinaris v0.7.2 is now available. 

Changes include:
- Drives monitoring page - allow [overrides of smartctl commands](https://github.com/guydavis/machinaris/wiki/Drives#override-command) for special hardware. Drive failure alerts thru our enhanced Chiadog.
 - Alerts: per-notifier [ignore setting](https://github.com/guydavis/machinaris/wiki/ChiaDog#ignore-alerts) to prevent unwanted alerts, based on criteria.
 - German translations of the Machinaris WebUI by @ApfelBirneKreis.  Big thanks!
 - [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.3.3) - v1.3.3 release, various minor bug fixes
 - [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.3.4) - v1.3.4 release, upgrade fullnode before harvesters
 - [Chia](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.3.5) - v1.3.5 release, fixes their `chia version` number offset
 - [Staicoin](https://github.com/STATION-I/stai-blockchain) - v1.2 release - careful, bugs reported on their Discord, but seems to be working...
  
Notes:
 - Chia Devs recommend updating fullnodes before updating harvesters when moving to latest Chia network.

<hr /><em>This discussion was created from the release <a href='https://github.com/guydavis/machinaris/releases/tag/0.7.2'>Version 0.7.2</a>.</em>

**Comment by Bafff** [comment]
So, harvesters or fullnode first? ))

![image](https://user-images.githubusercontent.com/26187677/168883875-6efe1de6-07d7-433e-adc1-4c71ed7e8ed4.png)

**Reply by guydavis** [reply]
Fullnodes before harvesters.  Typo corrected. Thanks.
